.github/workflows/update-base-images.yml

```diff
@@ -51,6 +52,7 @@ jobs:
           - { repo: "che-incubator/kubernetes-image-puller", user: "CHE_INCUBATOR_BOT_GITHUB_TOKEN", commandFlags: "-maxdepth 3 -f *Dockerfile --pr" }
           - { repo: "che-incubator/kubernetes-image-puller-operator", user: "CHE_INCUBATOR_BOT_GITHUB_TOKEN", commandFlags: "-maxdepth 3 -f *Dockerfile --pr" }
           - { repo: "devfile/devworkspace-operator", user: "CHE_INCUBATOR_BOT_GITHUB_TOKEN", commandFlags: "-maxdepth 3 -f *Dockerfile --pr" }
+          - { repo: "devfile/developer-images", user: "CHE_INCUBATOR_BOT_GITHUB_TOKEN", commandFlags: "-maxdepth 3 -f *Dockerfile --pr" }
```

Comment:
this will work if we omit updates for the second dockerfile, https://github.com/devfile/developer-images/pull/46/files#diff-9f21bef4669d474b6ddef409447be53ae91c94cb50b6df9345149a0e89fb5391R3

If we however want to figure out how to keep that one current... then we would need more flags here.

Reply:
note also that for this to work the secret has to be pushed to the repo. See step 3 on https://github.com/eclipse/che/issues/21575 (just added it tonight). I hope that you have permission to do that step?